README.md

```diff
@@ -0,0 +1,127 @@
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Budget App
+
+> This is an educational project is about building a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what. The Ruby on Rails application that allows the user to:
+- [x] register and log in, so that the data is private to them.
+- [x] introduce new transactions associated with a category.
+- [x] see the money spent on each category.
+
+The result should look similar to the following data model:
+
+<img width="374" alt="erd_diagram" src="https://user-images.githubusercontent.com/98527559/188287544-68d6fb5b-a265-4001-b1ac-3dc74393d720.png">
+
+## Learning Objectives 🔖
+- [x] Use ruby gems as software packages system.
+- [x] Install Ruby on Rails framework.
+- [x] Understand Rails RESTful design and router.
+- [x] Use controllers to handle requests and render empty views.
+- [x] Use params from browser request in a safe way.
+- [x] Use preprocessed html file with embedded Ruby code.
+- [x] Use layouts and templates for shared content.
+- [x] Use database migration files to maintain database schema.
+- [x] Use validations for models.
+- [x] Secure app from n+1 problems.
+- [x] Understand what ORM is.
+- [x] Write SQL queries with ActiveRecord.
+- [x] Set up associations between models.
+- [x] Build a webapp that requires the user to log in.
+- [x] Use devise gem for authentication.
+- [x] Limit access to webapp resources based on authorization rules.
+- [x] Analyze in writing why you have made a coding choice using one structure over another.
+
+## Project Documentation 📄
+
+- [x] Here is the presentation video [link](https://www.loom.com/share/41d82d0f57974b6589261248f0cd354d)👈
+- [x] Here is the deployed app in [Heroku](https://floating-stream-68134.herokuapp.com)👈
```

Comment:
- [ ] [optional] Kindly check on your deployed app it seems to have a bug that is slowing it down, check the deployment logs.

Reply:
- [ ] thank you for your suggestion but I think the speed of heroku  depend of the connection . :